cdc/sink/mysql_test.go

```diff
@@ -704,6 +704,29 @@ func (s MySQLSinkSuite) TestConfigureSinkURI(c *check.C) {
 		c.Assert(err, check.IsNil)
 		defer db.Close()
 
+		dsn, err := dmysql.ParseDSN("root:123456@tcp(127.0.0.1:4000)/")
+		c.Assert(err, check.IsNil)
+		params := defaultParams.Clone()
+		dsnStr, err := configureSinkURI(context.TODO(), dsn, params, db)
+		c.Assert(err, check.IsNil)
+		expectedParams := []string{
+			"tidb_txn_mode=optimistic",
+			"readTimeout=2m",
+			"writeTimeout=2m",
+			"allow_auto_random_explicit_insert=1",
+			"explicit_defaults_for_timestamp=ON",
```

Comment:
Could you test `on|OFF|off|1|0|❌|invalid` too?

Reply:
Check whether downstream is TiDB by `select version`, we don't relay the value of `explicit_defaults_for_timestamp` in database variable now. No need to verify whether these values are valid